<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>extTools</class>
 <widget class="QDialog" name="extTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QwtPlot" name="qwtPlot">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>551</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbWave">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>波形选择</string>
   </property>
   <widget class="QRadioButton" name="rbSin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>正弦波</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbSquare">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>方波</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbSaw">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>锯齿波</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbTri">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>三角波</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbCus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>自定义</string>
    </property>
   </widget>
  </widget>
  <widget class="QDoubleSpinBox" name="spVoltage">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>100</y>
     <width>62</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <double>28.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lbcon">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>135</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> &lt;--&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>380</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGenWData">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>280</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbPara">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>波形参数</string>
   </property>
   <widget class="QLabel" name="lbFEQ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FEQ:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbAMP">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>47</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1024.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbAMP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AMP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDUTY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DUTY:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbDUTY">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>167</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbTIME">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TIME:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbTIME">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>87</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>65536.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbamp_end">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;&quot;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckbloop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>97</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>循环产生</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnLoad">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>330</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string> 载入数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbvoltage">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>105</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电压量程:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbphy">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>70</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>物理量</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>对应关系:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbphy">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>165</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>物理量程:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbvoltage">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>电压</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spphysic">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>160</y>
     <width>62</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>721</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>自定义参数</string>
   </property>
   <widget class="QDoubleSpinBox" name="sbstartTime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>25</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>65536.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lblStartTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>StartTime:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbEndTime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>75</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>65536.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lblEndTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EndTime:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbStartY">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>25</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>StartY:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sbEndY">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>75</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EndY:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lswcurv">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>25</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirmSeg">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>115</y>
      <width>94</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加段</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncurDel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>94</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除段</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>-5</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> 已添加:</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>qwtPlot</zorder>
  <zorder>gbWave</zorder>
  <zorder>spVoltage</zorder>
  <zorder>lbcon</zorder>
  <zorder>btnClose</zorder>
  <zorder>btnGenWData</zorder>
  <zorder>gbPara</zorder>
  <zorder>btnLoad</zorder>
  <zorder>lbvoltage</zorder>
  <zorder>rbphy</zorder>
  <zorder>lbphy</zorder>
  <zorder>rbvoltage</zorder>
  <zorder>spphysic</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
